base/intset.jl

```diff
@@ -239,3 +245,8 @@ function hash(s::IntSet, h::UInt)
     end
     h
 end
+
+minimum(s::IntSet) = first(s)
+maximum(s::IntSet) = last(s)
+extrema(s::IntSet) = (first(s), last(s))
+issorted(s::IntSet) = true
```

Comment:
Is the order a set is iterated in even defined i. e does the order have to be the same every time even if the set is unchanged? Defining `issorted` for something unordered feels a  bit iffy and perhaps mixes implementation with public API.